pkg/builder/builder.go

```diff
@@ -250,14 +251,67 @@ func loadArchive(ctx *Context) (err error) {
 	return nil
 }
 
+// Merges source and destination map, preferring values from the source map
+func mergeValues(dest map[string]interface{}, src map[string]interface{}) map[string]interface{} {
```

Comment:
cc @bacongobbler, @michelleN 
Are there any other projects you are aware of that re-implement the merging of value files from Helm?

Would it make sense to expose this in the Helm `pkg`?

Reply:
Google's `skaffold`, for example, just [calls `helm` binary directly](https://github.com/GoogleContainerTools/skaffold/blob/3c275201e8e1f7cf4522897394467aa8f6bc34c6/pkg/skaffold/deploy/helm.go#L109-L112) and [passes all `valuesFiles` as arguments](https://github.com/GoogleContainerTools/skaffold/blob/3c275201e8e1f7cf4522897394467aa8f6bc34c6/pkg/skaffold/deploy/helm.go#L213-L215) to it, i.e. it doesn't deal with merging them at all — but even if this is the first occurrence, I think it makes sense to expose this in the Helm.

Reply:
Should I make a PR for helm? Would `chartutil` be the right package for it?

Reply:
I'm pretty sure this already exists in `pkg/chartutil` called `MergeInto`: https://github.com/helm/helm/blob/403af2c3895e4fa0e04713d94135bf3438d9ce9c/pkg/chartutil/values.go#L97-L111

Probably would be good to use that instead

Reply:
Usage: https://github.com/helm/helm/blob/82d01cb3124906e97caceb967a09f2941d6a392d/pkg/chartutil/values_test.go#L521-L540

Reply:
From what I could see, parts where supported by `chartutil` already, but not other parts, will take a look at refining it, whether it's in Helm or here.